adding section telling user to change column type to numeric prior to summarizing (#174)
gristlabs/grist-help: help/investment-research.md

## Summarize

> The powerful feature you’ve been waiting for is the one that summarizes the data. [Summary tables](summary-tables.md) summerize each numeric column in a data table. We want to find the sum for the funding_total_usd column in our Companies table. Set the column type to 'Numeric' and change the Number Format to `$`.

Reviewer: ```suggestion
The powerful feature you’ve been waiting for is the one that summarizes the data. [Summary tables](summary-tables.md) summarize each numeric column in a data table. We want to find the sum for the `funding_total_usd` column in the Companies table. Check that the column type is set to 'Numeric' and and formatted with `$`.
```
